i18n/locales/es-MX/dashboard/blocks.yml

```diff
@@ -553,7 +553,7 @@ es-MX:
             '"ROBOT"': todos los robots
             '"SHARK"': todos los tiburones
       Dancelab_setTintInline:
-        text: definir tinte [SPRITE] [0] a [VAL] [1]
+        text: definir tinte {SPRITE} a {VAL}
```

Comment:
hmm this doesn't look right but it's also not what's in [crowdin](https://crowdin.com/translate/codeorg/639/enus-esmx#q=Dancelab_setTintInline)

Reply:
No, this is right. The stuff that's on crowdin is the redacted version; things like `[SPRITE] [0] a [VAL] [1]` are what show up when the translator adds spaces. This change is reflective of the errant spaces being removed from the translation, so the restoration process works again.

Reply:
Great thanks for the explanation!